.github/workflows/helm-chart-lint.yaml

```diff
@@ -41,20 +51,22 @@ jobs:
         run: |
           changed=$(ct list-changed --target-branch ${{ github.event.repository.default_branch }})
           if [[ -n "$changed" ]]; then
-            echo "CHART_CHANGED=true" >> $GITHUB_ENV
+            echo "changed=true" >> $GITHUB_OUTPUT
           fi
 
-      # run chart linting
       - name: Run chart-testing (lint)
-        run: ct lint --target-branch ${{ github.event.repository.default_branch }} --config charts/config/chart-testing-config.yaml
-
-      # Preparing a kind cluster to install and test charts on
-      - name: Create kind cluster
-        uses: helm/kind-action@v1.4.0
-        if: ${{ env.CHART_CHANGED == 'true' }}
-
-      # install the chart to the kind cluster and run helm test
-      # define charts to test with the --charts parameter
+        run: ct lint --validate-maintainers=false --target-branch ${{ github.event.repository.default_branch }}
```

Comment:
here please also include parameter --config charts/config/chart-testing-config.yaml

(then you don't need --validate-maintainers=false because its already included in the config)